legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py

```diff
@@ -1703,3 +1703,17 @@ def test_temp_document_list_for_now(mocker, session, client, jwt,
 
     assert rv.status_code == expected_http_code
     assert rv_data == expected
+
+    filing._status = Filing.Status.PAID
```

Comment:
this should be `COMPLETED` right?

Also, the expected documents returned are the same for both `PAID` and `COMPLETED` status?